srml/assets/src/lib.rs

```diff
@@ -145,7 +147,7 @@ decl_module! {
 			let origin = ensure_signed(origin)?;
 
 			let id = Self::next_asset_id();
-			<NextAssetId<T>>::mutate(|id| *id += 1);
+			<NextAssetId<T>>::mutate(|id| *id += 1.into());
```

Comment:
```suggestion
			<NextAssetId<T>>::mutate(|id| *id += One::one());
```

You will need to import `One`.

Reply:
it's more explicit way. Thanks.